src/Simplex/Chat/View.hs

```diff
@@ -198,17 +198,21 @@ responseToView hu@(currentRH, user_) ChatConfig {logLevel, showReactions, showRe
   CRGroupMemberUpdated {} -> []
   CRContactsMerged u intoCt mergedCt ct' -> ttyUser u $ viewContactsMerged intoCt mergedCt ct'
   CRReceivedContactRequest u UserContactRequest {localDisplayName = c, profile} -> ttyUser u $ viewReceivedContactRequest c profile
-  CRRcvFileStart u ci -> ttyUser u $ receivingFile_' hu testView "started" ci
-  CRRcvFileComplete u ci -> ttyUser u $ receivingFile_' hu testView "completed" ci
+  CRRcvFileStart u ci ft -> ttyUser u $ receivingFile_' hu testView "started" (maybe (Left ft) Right ci)
+  CRRcvFileComplete u ci -> ttyUser u $ receivingFile_' hu testView "completed" (Right ci)
+  CRRcvFileCompleteXFTP u _ ft -> ttyUser u $ receivingFile_' hu testView "completed" (Left ft)
   CRRcvFileSndCancelled u _ ft -> ttyUser u $ viewRcvFileSndCancelled ft
-  CRRcvFileError u ci e -> ttyUser u $ receivingFile_' hu testView "error" ci <> [sShow e]
+  CRRcvFileError u ci e ft -> ttyUser u $ receivingFile_' hu testView "error" (maybe (Left ft) Right ci) <> [sShow e]
```

Comment:
```suggestion
  CRRcvFileError u ci e ft -> ttyUser u $ maybe (receivingStandaloneFile ft) receivingFile_' ci hu testView "error" <> [sShow e]
```

split to two functions, ci will be the first parameter, the rest can repeat (if they are needed in receivingStandaloneFile)